app/components/UI/SimulationDetails/BalanceChangeRow/BalanceChangeRow.tsx

```diff
@@ -10,17 +10,27 @@ import { BalanceChange } from '../types';
 import AmountPill from '../AmountPill/AmountPill';
 import AssetPill from '../AssetPill/AssetPill';
 import { IndividualFiatDisplay } from '../FiatDisplay/FiatDisplay';
+import EditRowValue from '../EditRowValue';
 import styleSheet from './BalanceChangeRow.styles';
 
 interface BalanceChangeRowProperties extends ViewProps {
   label?: string;
   showFiat?: boolean;
   balanceChange: BalanceChange;
+  enableEdit?: boolean;
+  onUpdate?: (balanceChange: BalanceChange, val: string) => void;
+  editTexts?: {
+    title: string;
+    description: string;
+  };
```

Comment:
```suggestion
  balanceChange: BalanceChange;
  editTexts?: {
    title: string;
    description: string;
  };
  enableEdit?: boolean;
  label?: string;
  onUpdate?: (balanceChange: BalanceChange, val: string) => void;
  showFiat?: boolean;
```
Minor, alphabetical?